app/components/job/action/add/job-create-basic-dialog.component.ts

```diff
@@ -69,6 +70,31 @@ export class JobCreateBasicDialogComponent extends DynamicForm<Job, JobCreateDto
         this.form.controls.jobPreparationTask.valueChanges.subscribe((value) => {
             this.showJobReleaseTask = value && value.id;
         });
+
+        // Load current pool container configuration to indicate whether job manager, preparation and release task
+        // displays task container setting accordingly
+        this.form.controls.poolInfo.valueChanges.subscribe((value) => {
+            if (poolService && poolService.get && value && value.poolId) {
+                poolService.get(value.poolId).cascade((pool) => {
```

Comment:
so here I wouldn't use a subscribe on valueChanges but more of a flatMap with debounceTime so it will cancel the previous request if you click on multiple fast instead of loading all of them and potentially setting the wrong value if the first request finish after the second